gui/src/main/java/org/gui/App.java

```
@@ -54,17 +59,43 @@ public void start(Stage stage) {
 		stage.show();
 	}
 
-
 	public AppMenu getMenu() {
 		return menu;
 	}
 
+	public static void setLocale(Locale locale1) {
+
+		try (FileWriter writer = new FileWriter("lang.txt", false)) {
+			String text = locale1.toString();
+			writer.write(text);
+			writer.flush();
+		} catch (IOException ex) {
```

Empty catch is not good